src/ILCompiler.MetadataTransform/tests/SimpleTests.cs

```diff
@@ -39,29 +39,6 @@ public void TestAllTypes()
                 transformResult.Scopes.Single().GetAllTypes().Count());
         }
 
-        [Fact]
```

Comment:
Please keep this test. We can rename ICastable to IBlockedInterface or something like this, but this is only testing that blocked interfaces don't show up in interface lists. This is a situation that can still happen with different interfaces and we should keep the unit test coverage.

Reply:
(Basically, all occurences under ILCompiler.MetadataTransform are fine)